### ISSUE TITLE
⌨ (In progress) Accessibility - Include a submit button for the profile status radio buttons

### ISSUE THREAD

**Issue by MaddyDaigle** [GCTC-NTGC/gc-digital-talent#5003]
**In progress - don't dev!!!**

## ❌ Current behaviour (issue)
Currently, if the user interacts with any of the radio buttons on the profile status section of the profile page, the entire page is refreshed.  There is no feedback regarding what is occurring for users.  This is especially jarring for screen reader and keyboard users because they are bumped up to the top of the page on refresh.

**WCAG 2.1 Criteria:**

### How to reproduce
1. Environment (prod or UA?) Prod
2. Which browser and/or Assistive Technology? Chrome
3. Step-by-step instructions explaining how to reproduce the problem.

## ✔ Expected behaviour (fix)
Include an "update my status" button after the radio buttons so that the user understands they are submitting a form and an action is occurring.

To determine: Should there be a disabled button if the user cannot update their profile?  Or should the button only show up if the user has completed their profile?

### Acceptance Criteria
A set of assumptions which, when tested, verify that the accessibility requirement was met.
- [ ] Criteria
- [ ] Criteria 2

## Additional information (optional)
1. How does it impact users?
2. Related issues, suggestions how to fix

**Comment by esizer** [comment]
@petertgiles I believe we have removed this form entirely, can we close this issue now?

**Comment by petertgiles** [comment]
Oh yeah, hidden in  #5929.  Technically still visible until we flip the feature flag but we shouldn't make any non-urgent changes to it now.
